ci/continuous/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM quay.io/coreos-assembler/fcos:testing-devel
```

```suggestion
FROM quay.io/coreos-assembler/fcos:testing-devel
```

I assume config bot is going to copy these files to the other branches. We might want to think about how this line translates to the the other branches.